### ISSUE TITLE
OAuth login using Firebase

### ISSUE THREAD

**Issue by ikjot-2605** [WebClub-NITK/Hacktoberfest-2k20#3]
### Description
Create a web/mobile application that allows a user to sign in using his/her (Google account/Phone Number).
You are recommended to use the Authentication feature from Firebase.

### Details
- Technical Specifications: Web(HTML/CSS/JS) or App(Flutter(Recommended)/Android) Development + Firebase
- Type of issue: Single
- Time Limit: 48 hours

### Issue requirements / progress
- [ ] Implement OAuth using Firebase

### Resources
https://firebase.google.com/docs/auth/web/custom-auth
https://firebase.google.com/docs/flutter/setup   (Recommended)
### Directory Structure
Create a new folder with the name "3_OAuth_Login" inside the development folder and upload all the relevant files inside it.

#### Note
Please claim the issue first by commenting here before starting to work on it.
Please contact @ikjot-2605 for any issues/doubts/questions regarding the same.

**Comment by vnishchay** [comment]
I'm want to work on it.

**Comment by ikjot-2605** [comment]
Sure, go ahead.

**Comment by chakravarthi-v** [comment]
@ikjot-2605 if it possible I want to do it

**Comment by DineshBS44** [comment]
Can only 1 person work on 1 issue? If not, can I also contribute to this? I'm an Android developer.
.

**Comment by ikjot-2605** [comment]
@DineshBS44  and @chakravarthi-v  This issue was already assigned, since @codernishchay had claimed it before you.
It shall be assigned to one of you if @codernishchay fails to complete it within the given time limit(3rd October, 2:52 am). Until then please explore other issues on the repo.

**Comment by ikjot-2605** [comment]
You can take it up now @DineshBS44 , @chakravarthi-v. Whoever can claim it by commenting, can start working on it.

**Comment by DineshBS44** [comment]
@ikjot-2605, I would like to work on this.

**Comment by ikjot-2605** [comment]
Go ahead @DineshBS44 !

**Comment by DineshBS44** [comment]
Could you please add the Hacktoberfest label to this issue?

**Comment by DineshBS44** [comment]
@ikjot-2605, I made a Pull Request, please check!

**Comment by ikjot-2605** [comment]
Sure @DineshBS44 I'll look through your PR and suggest any changes if required.

**Comment by DineshBS44** [comment]
@ikjot-2605, kindly try to review the PR by tomorrow morning because I have a deadline of submitting another PR for tomorrow afternoon in the same repo.

**Comment by ikjot-2605** [comment]
@krithikvaidya I have reviewed the MR and it's ready to be merged.

**Comment by DineshBS44** [comment]
@krithikvaidya , could you please add "hacktoberfest-accepted" label to this issue? It is mandatory for the PR count as the new rules in the hacktoberfest website states.

**Comment by krithikvaidya** [comment]
Hey @DineshBS44 , since our repository has the hacktoberfest topic and your PR is merged, there is no need of the hacktoberfest-accepted label, it'll count towards hacktoberfest 👍

**Comment by DineshBS44** [comment]
@krithikvaidya, okay understood. Thanks for informing!

**Comment by krithikvaidya** [comment]
@ikjot-2605 I'll close the issue since the PR has been merged.
